### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by YuliaParasinchuk**
Add operations and parser in the CalculatorCore library

### CODE REVIEW

File: Calculator/CalculatorCore/Parser.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public class Parser
+    {
+        static Regex rgx = new Regex(@"^-?\d[.,]\d+");
+
+        public static double GetDoubleNumberFromString(string str)
+        {   
+            if (rgx.IsMatch(str))
+            {
+                str = str.Replace(".",",");
```
Comment (bidolah) [inline Calculator/CalculatorCore/Parser.cs]: Please use something like this:
`Double.TryParse(String, NumberStyles, IFormatProvider, Double)`

File: Calculator/CalculatorCore/Operations.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            return firstNumber * secondNumber;
+        }
+
+        public static double Division(double firstNumber, double secondNumber)
+        {
+            if(secondNumber != 0)
+                return firstNumber / secondNumber;
+           // return Convert.ToDouble((double?)null);
+            return double.NaN;
```
Comment (bidolah) [inline Calculator/CalculatorCore/Operations.cs]: Use nullable type "double?" or "int?" for all function in this class

File: Calculator/CalculatorConsole/Program.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+using CalculatorCore;
+
+
+namespace CalculatorConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(Operations.Sum(1.25, 2.2));
```
Comment (bidolah) [inline Calculator/CalculatorConsole/Program.cs]: Make console output more beautiful. Example for sum operation `1.25 + 2.2 = 3.45`
Create output test for all functions in Core library

**Comment by YuliaParasinchuk** [comment]
Make changes

**Pull request merged**